vizro-core/src/vizro/_themes/create_chart_templates.py

```diff
@@ -0,0 +1,137 @@
+
+    # Apply common values
+    template = create_template_common()
+    template.layout.font.color = FONT_COLOR_PRIMARY
+    template.layout.title.font.color = FONT_COLOR_PRIMARY
+    template.layout.legend.font.color = FONT_COLOR_PRIMARY
+    template.layout.legend.title.font.color = FONT_COLOR_PRIMARY
+    template.layout.paper_bgcolor = BG_COLOR
+    template.layout.plot_bgcolor = BG_COLOR
+    template.layout.geo.bgcolor = BG_COLOR
+    template.layout.geo.lakecolor = BG_COLOR
+    template.layout.geo.landcolor = BG_COLOR
+    template.layout.polar.bgcolor = BG_COLOR
+    template.layout.polar.angularaxis.gridcolor = GRID_COLOR
+    template.layout.polar.angularaxis.linecolor = AXIS_COLOR
+    template.layout.polar.radialaxis.gridcolor = GRID_COLOR
+    template.layout.polar.radialaxis.linecolor = AXIS_COLOR
+    template.layout.ternary.bgcolor = BG_COLOR
+    template.layout.ternary.aaxis.gridcolor = GRID_COLOR
+    template.layout.ternary.aaxis.linecolor = AXIS_COLOR
+    template.layout.ternary.baxis.gridcolor = GRID_COLOR
+    template.layout.ternary.baxis.linecolor = AXIS_COLOR
+    template.layout.ternary.caxis.gridcolor = GRID_COLOR
+    template.layout.ternary.caxis.linecolor = AXIS_COLOR
+    template.layout.mapbox.style = "carto-darkmatter"
+    template.layout.coloraxis.colorbar.tickcolor = AXIS_COLOR
+    template.layout.coloraxis.colorbar.tickfont.color = FONT_COLOR_SECONDARY
+    template.layout.coloraxis.colorbar.title.font.color = FONT_COLOR_SECONDARY
+    template.layout.xaxis.title.font.color = FONT_COLOR_PRIMARY
+    template.layout.xaxis.tickcolor = AXIS_COLOR
+    template.layout.xaxis.tickfont.color = FONT_COLOR_SECONDARY
+    template.layout.xaxis.linecolor = AXIS_COLOR
+    template.layout.xaxis.gridcolor = GRID_COLOR
+    template.layout.yaxis.title.font.color = FONT_COLOR_PRIMARY
+    template.layout.yaxis.tickcolor = AXIS_COLOR
+    template.layout.yaxis.tickfont.color = FONT_COLOR_SECONDARY
+    template.layout.yaxis.linecolor = AXIS_COLOR
+    template.layout.yaxis.gridcolor = GRID_COLOR
+    template.layout.annotationdefaults.font.color = FONT_COLOR_PRIMARY
+
+    # "map" only available in plotly>=5.24.0, will replace "mapbox" soon. Until then, we need to set both.
+    # We need the if statement here in case the user is using an older version of plotly.
+    if "map" in template["layout"]:
+        template.layout.map.style = "carto-darkmatter"
+
+    template.data.bar = [go.Bar(marker_line_color=BG_COLOR)]
+    template.data.waterfall = [
+        go.Waterfall(
+            decreasing={"marker": {"color": COLORS["DISCRETE_10"][1]}},
+            increasing={"marker": {"color": COLORS["DISCRETE_10"][0]}},
+            totals={"marker": {"color": "grey"}},
+            textfont_color=FONT_COLOR_PRIMARY,
+            textposition="outside",
+            connector={"line": {"color": AXIS_COLOR, "width": 1}},
+        )
+    ]
+    return template.to_plotly_json()
```

Comment:
```suggestion
    return template
```
I feel like this is unnecessary given the `json.dumps` in `__main__`?